docs/5.0/preview/database-access.md

```diff
@@ -135,7 +216,7 @@ auth_service:
   cluster_name: "test"
   listen_addr: 0.0.0.0:3025
   tokens:
-  - proxy,node,database:qwe123
+  - proxy,node,database:cbdeeab9-6f88-436d-a673-44d14bd86bb7
```

Comment:
nit: I know it must have been suggested to use token that resembled a real token, but since this needs to be matched in multiple commands and yet be easier to compare/remember when skimming through the docs - how about using something more memorable like `00000000-0000-0000-0000-000000000001`?

Reply:
I think using a real uuid sends a stronger signal to readers that using a secure token is advised.